samples/Sentry.Samples.AspNetCore.Basic/Program.cs

```diff
@@ -70,5 +73,21 @@ public static void Main(string[] args)
                     });
                 })
                 .Build();
+
+        public static IHost BuildHost(string[] args) =>
+            Host.CreateDefaultBuilder(args)
+
+                // Add Sentry integration
+                // In this example, DSN and Release are set via environment variable:
+                // See: Properties/launchSettings.json
+                .UseSentry()
+                // It can also be defined via configuration (including appsettings.json)
+                // or coded explicitly, via parameter like:
+                // .UseSentry("dsn") or .UseSentry(o => o.Dsn = ""; o.Release = "1.0"; ...)
+                .ConfigureWebHost(webBuilder =>
+                {
+                    webBuilder.UseStartup<Startup>();
```

Comment:
What about this? Sometimes I find in samples it's good to be very explicit about things:

```suggestion
                    // Note that it's NOT needed to also call UseSentry here. That's because it's already hooked through IHostBuilder
                    webBuilder.UseStartup<Startup>();
```